### PR TITLE
Upgrade unicorn to play 2.5, introduce DI, remove Play.current #82 #75

### PR DESCRIPTION
**Pull request by liosedhel**
Upgrade unicorn to play 2.5, introduce DI, remove Play.current #82 #75

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 language: scala
 scala:
-  - "2.10.4"
-  - "2.11.5"
+  - "2.11.7"
```
Comment (khar) [inline .travis.yml]: 2.11.8

File: project/Settings.scala [hunk 2]
```diff
@@ -14,10 +13,8 @@ object Settings {
   // settings for ALL modules, including parent
   val core = Seq(
     organization := "org.virtuslab",
-    scalaVersion := "2.11.6",
-    crossScalaVersions := Seq("2.10.4", scalaVersion.value),
-    ReleaseKeys.crossBuild := true,
-    ReleaseKeys.publishArtifactsAction := PgpKeys.publishSigned.value,
+    scalaVersion := "2.11.7",
```
Comment (khar) [inline project/Settings.scala]: 2.11.8

**Comment by kentongray** [comment]
hate to be the one who asks but what is the status of this PR?

**Comment by liosedhel** [comment]
Oh, it is still not merged, my oversight, sorry :( I had some positive response here: #82. Is it working for you also @kentongray ? (There were some doubts about current API). I will check this PR again and ask for merge.

**Comment by liosedhel** [comment]
Commits have been squashed. Can we merge this PR @Kwestor ? I've also prepared some POC of the Lightbend activator template with this new version of Unicorn: https://github.com/VirtusLab/activator-play-advanced-slick/pull/8.

**Comment by Kwestor** [comment]
Merged.

**Pull request merged**